src/discord/commands/NetworthCommand.js

```diff
@@ -0,0 +1,87 @@
+/*
+ CREDIT: Idea by Aura
+ Discord: Aura#5051
+ Minecraft username: _aura
+*/
+const {SlashCommandBuilder} = require('@discordjs/builders')
+const fetch = require("axios");
+
+const MARO_ENDPOINT = "https://skyblock.acebot.xyz/api/networth/categories"
+const HYPIXEL_KEY = process.env.HYPIXEL_KEY
+
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('networth')
+        .setDescription('Search someone\'s networth.')
+        .addStringOption(option =>
+            option.setName('username')
+                .setDescription('User name')
+                .setRequired(true)),
+    permission: 0, // 0 = anyone, 1 = staff, 2 = owner/admin
+
+    async execute(clientInstance, interaction) {
+        await interaction.deferReply()
+
+        let uuid = await getUuidByUsername(username)
+
+        if (!uuid) {
+            interaction.editReply(`No such username! (given: ${username})`)
```

Comment:
add blank line after end of if.
```suggestion
            interaction.editReply(`No such username! (given: ${username})`)
            
```